SPECS/tdnf/tdnf.spec

```
@@ -5,7 +5,7 @@
 Summary:        dnf/yum equivalent using C libs
 Name:           tdnf
 Version:        2.1.0
-Release:        7%{?dist}
+Release:        8%{?dist}
 License:        LGPLv2.1 AND GPLv2
```

Mind changing this to use an [SPDX license expression](https://spdx.org/licenses/), while you're here?

```suggestion
License:        LGPL-2.1-only AND GPL-2.0-only
```

---

Sure thing!